src/Features/CSharp/Portable/CodeFixes/PreferFrameworkType/CSharpPreferFrameworkTypeCodeFixProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.PreferFrameworkType;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.PreferFrameworkType
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.PreferFrameworkType), Shared]
+    internal class CSharpPreferFrameworkTypeCodeFixProvider : AbstractPreferFrameworkTypeCodeFixProvider
+    {
+        protected override SyntaxNode GenerateTypeSyntax(ITypeSymbol symbol) => symbol.GenerateTypeSyntax();
```

Comment:
Ideally, if VB has the same impl, then we could move this to something like ISemanticFactsService.  Then you wouldn't need any subclassing at all.